omnibus/package-scripts/chefdk/postrm

```diff
@@ -25,7 +25,7 @@ else
 fi
 
 cleanup_symlinks() {
-  binaries="chef chef-solo chef-apply chef-shell knife shef ohai berks chef-zero fauxhai foodcritic kitchen rubocop strain strainer chef-client"
```

Comment:
That's what I'd want it to do, for sure, though I think I remember this not always being the case. Either way, we can probably ignore that for now.